dev/merge_spark_pr.py

```diff
@@ -495,7 +495,24 @@ def main():
     else:
         title = pr["title"]
 
-    body = pr["body"]
+    modified_body = re.sub(re.compile(r'<!--[^>]*-->\n?', re.DOTALL), '', pr["body"]).lstrip()
```

Comment:
`lstrip()` is used to remove the first PR template comments and new lines. `strip()` has false positive because it will modify all PR description (which doesn't have comments).